### ISSUE TITLE
Grouping layers does not get a new layer event

### ISSUE THREAD

**Issue by iwehrman** [adobe-photoshop/generator-core#306]
As reported here: https://groups.google.com/forum/#!topic/adobe-generator-dev/2GtcxgZUapw

---

When you create a group by grouping layers (select layer, press Ctrl+G), the event does not have an "added=true" parameter for the new group. Instead it looks like the layer was moved to an existing group like below

```javascript
  "version":"1.1.0",
  "timeStamp":1433319347.348,
  "count":8,
  "id":1118,
  "changed":true,
  "layers":[{
    "id":3,
    "index":7,
    "layers":[{
      "id":8,
      "index":4,
      "bounds":{
        "top":0,
        "left":0,
        "bottom":0,
        "right":0
        },
      "layers":[{
        "id":5,
        "index":3
        }]
      }]
    }],
```

Steps to replicate (Photoshop CC 2014 latest version)

1. Create a new document
2. Select layer
3. Hit Ctrl-G to group the layer

Also the event is missing the name and type of the new group

**Comment by joelrbrandt** [comment]
User reports issue is still present in CC 2015.

@jaredadobe any chance this is the same issue as #307 ? If so, can I sucker you into fixing them both at the same time?

Thanks!

**Comment by joelrbrandt** [comment]
Filed Watson bug 4036322

**Comment by timothynoel** [comment]
Fixed for 16.1

**Comment by joelrbrandt** [comment]
User indeed reports this is fixed in 2015.1.1. Thanks @timothynoel
